samples/snippets/fsharp/azure/queue-storage.fsx

```diff
@@ -12,102 +11,87 @@ let storageConnString = "..." // fill this in from your storage account
 let storageConnString = 
     CloudConfigurationManager.GetSetting("StorageConnectionString")
 *)
-//
-// Parse the connection string.
-//
-
-// Parse the connection string and return a reference to the storage account.
-let storageAccount = CloudStorageAccount.Parse(storageConnString)
 
 //
 // Create the Queue Service client.
 //
 
-let queueClient = storageAccount.CreateCloudQueueClient()
+let queueClient = QueueClient(storageConnString, "myqueue");
 
 //
 // Create a queue.
 //
 
-// Retrieve a reference to a container.
-let queue = queueClient.GetQueueReference("myqueue")
-
-// Create the queue if it doesn't already exist
-queue.CreateIfNotExists()
+queueClient.CreateIfNotExists()
 
 //
 // Insert a message into a queue.
 //
 
-// Create a message and add it to the queue.
-let message = new CloudQueueMessage("Hello, World")
-queue.AddMessage(message)
+queueClient.SendMessage("Hello, World")
 
 //
 // Peek at the next message.
 //
 
-// Peek at the next message.
-let peekedMessage = queue.PeekMessage()
-let msgAsString = peekedMessage.AsString
+let peekedMessage = queueClient.PeekMessage()
+let messageContents = peekedMessage.ToString()
```

Comment:
Have you tested this?  We don't override `ToString` so I don't think you'll get anything good out here.  At a minimum, I think you want `peekedMessage.Value.Body.ToString()`.